### PR TITLE
refactor(converter-ms-sems): Use snapshot testing

### PR DESCRIPTION
**Pull request by jonahkagan**
## Overview
Instead of having manually created snapshot files, use a snapshot library. Otherwise it's impossible to change any logic without having to manually recreate expected test results.

## Checklist
- [ ] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.
- [ ] I have added JSDoc comments to any newly introduced exports

- [ ] I have added a screenshot and/or video to this PR to demo the change
- [ ] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates

### CODE REVIEW

**Review by adghayes — APPROVED** [review]

File: services/converter-ms-sems/tests/test_SEMSoutput.py [hunk 1]
```diff
@@ -1,49 +1,51 @@
...
+
+        TESTS = [
+            {
+                'election': get_sample_file('general-election.json'),
+                'cvrs': '10_cvrs.csv',
+                'tallies': '10_tallies.json',
+            },
+            {
+                'election': election_file.name,
+                'cvrs': '10_8-26-2020-cvrs.txt',
```
Comment (adghayes) [inline services/converter-ms-sems/tests/test_SEMSoutput.py:31]: Where are we using these `cvrs` files?

Comment (jonahkagan) [reply services/converter-ms-sems/tests/test_SEMSoutput.py:31]: Uhh, that's weird. They were there before but also unused. I guess I can remove?

Comment (adghayes) [reply services/converter-ms-sems/tests/test_SEMSoutput.py:31]: If it doesn't break anything, I think so. The test files in are in our current CVR file format, which we'll be changing anyway, so no use keeping around.

I didn't think we convert directly from our CVRs - we convert from an overall tally when we export results - so I'm not sure what those are doing there.

Comment (jonahkagan) [reply services/converter-ms-sems/tests/test_SEMSoutput.py]: Maybe an artifact from an earlier version of the script. Removed in 2da763a53

**Pull request merged**